src/APIs/APIManager.ts

```diff
@@ -52,7 +52,7 @@ class APIManager {
         })
     }
     async searchBooks(key:string) {
-        const res = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=${key}`)
+        const res = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=${key}&maxResult=40`)
```

Comment:
Why not add another argument to the `searchBooks` function called "params" which you will then add to the request in a way which is more generic? For example: 

`const baseURL = "https://www.googleapis.com/books/v1";`

and then inside `searchBooks`: 

```
searchBooks(key: string, params: GetVolumesParams) { // Create a type called "GetVolumesParams"
const config = {
  baseURL,
  url: '/volumes',
  params
}
const res = await axios.get(booksApiURL, config);
```

See: [axios config](https://axios-http.com/docs/req_config)